Restore :current pseudo class (#23242)
mdn/content: files/en-us/web/css/_colon_current/index.md

---
title: ':current'
slug: Web/CSS/:current
page-type: css-pseudo-class
browser-compat: css.selectors.current
---
{{CSSRef}}

> The **`:current`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) selector is a time-dimensional pseudo-class that represents the element, or an ancestor of the element, that is currently being displayed. For example in a video with captions which are being displayed by [WebVTT](/en-US/docs/Web/API/WebVTT_API).

Reviewer: ```suggestion
The **`:current`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) selector is a time-dimensional pseudo-class that represents the element or an ancestor of the element, that is currently being displayed. For example in a video with captions which are being displayed by [WebVTT](/en-US/docs/Web/API/WebVTT_API).
```

Reviewer: The phrase 'element or an ancestor of the element' are elements in a series. Elements in a series only require commas when there are three or more. Also, 'or an ancestor of the element' is restrictive and therefore should have no comma.

(I had to dig out the grammar book on this one. Let me know if I've overlooked anything.)

Reviewer 2: That is the correct rule for comma with conjunctions! 🙂

> The **`:current`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) selector is a time-dimensional pseudo-class that represents the element, or an ancestor of the element, that is currently being displayed. For example in a video with captions which are being displayed by [WebVTT](/en-US/docs/Web/API/WebVTT_API).

Reviewer 2: ```suggestion
The **`:current`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) selector is a time-dimensional pseudo-class that represents an element or the ancestor of an element that is currently being displayed. For example, this pseudo-class can be used to represent a video that is being displayed with captions by [WebVTT](/en-US/docs/Web/API/WebVTT_API).
```
also dropping the comma before 'that'
